### DISCUSSION TITLE
New badges

### DISCUSSION THREAD

**Discussion by muhammadashariabdillah24** [community/community#20700] [Other Feature Feedback, Questions, & Ideas]
Wow, i'm very happy , but i only just get badges from github that is Pull shark anda Yolo ... I want to say Thank very much :)

**Comment by radar-blip** [comment]
You can get more badges if you try. Here is a link to help you:

https://github.com/Schweinepriester/github-profile-achievements#achievements
Good Luck!

**Comment by pragusga25** [comment]
You can get Galaxy Brain badge too by answering at least 2 discussions in public repository. I think that badge is really cool.
